addons/crm_livechat/__manifest__.py

```diff
@@ -15,4 +15,9 @@
     'description': 'Create new lead with using /lead command in the channel',
     'auto_install': True,
     'license': 'LGPL-3',
+    'assets': {
+        'web.assets_backend': [
+            'crm_livechat/static/src/models/*/*.js',
```

Comment:
This follows the current structure that we have for our files, it's always exactly how I described it.
We don't want to support having files directly in models/ and we don't want subfolders either because that's where we currently put test files. Unless there is a way to make a negative glob to exclude tests on a second line, we don't want `**`.

Reply:
You can, in fact, use `('remove', glob)` to explicitly exclude some files, but if the file structure is consistent with the glob I don't see any issue with it.

Reply:
Let's keep it like this then, we can always change it if/when we change the structure.